### DISCUSSION TITLE
forget password return null in responce body instead of token ?

### DISCUSSION THREAD

**Discussion by udayzee05** [fastapi-users/fastapi-users#1336] [Q&A]
## Configuration
 - Python version :
 - FastAPI version :
 - FastAPI Users version :

### FastAPI Users configuration

```py
# Please copy/paste your FastAPI Users configuration here.
```

**Comment by frankie567** [comment]
https://fastapi-users.github.io/fastapi-users/12.1/usage/routes/#post-forgot-password
